int16 setpoint
int16 state
int16 effort
float32 longtitude
float32 latitude
